src/parser/jobs/brd/About.js

```diff
@@ -0,0 +1,27 @@
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+import {ActionLink} from 'components/ui/DbLink'
+
+import CONTRIBUTORS from 'data/CONTRIBUTORS'
+import CoreAbout from 'parser/core/modules/About'
+import ACTIONS from 'data/ACTIONS'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
+		<p>Welcome to the Bard module! Despite being a very straightforward job, Bard's complexity is deceiving.</p>
+		<p>Considered by many as an <i>"easy to learn, hard to master"</i> job, Bard is a job that relies heavily on decision-making.</p>
+		<p>Improvements on Bard can range from the fundamentals of properly utilizing songs (<ActionLink {...ACTIONS.THE_WANDERERS_MINUET}/>, <ActionLink {...ACTIONS.MAGES_BALLAD}/> and <ActionLink {...ACTIONS.ARMYS_PAEON}/>) up to the intricacies of <ActionLink {...ACTIONS.IRON_JAWS}/> and the concept of buff/debuff snapshotting.</p>
+		<p>This analyzer will guide you through the job's core mechanics, all the way to encounter-specific optimization.</p>
+
+		<Message warning icon>
```

Comment:
Nice description, I really like it!

Can I just get a copy update for the warning please, I think it's still got a bit of a ways to go 😓

Reply:
Uh, I think I changed this one? It's different in my working copy huh